.expeditor/config.yml

```diff
@@ -22,3 +22,6 @@ subscriptions:
   - workload: artifact_published:stable:inspec:*
     actions:
       - bash:.expeditor/update_hugo_modules.sh
+  - workload: artifact_published:stable:automate:*
```

Comment:
The "stable" channel for Automate is actually `current`. Additionally Automate does not promote an Omnibus package so the workload we need to subscribe to is slightly different:
```
# Subscribe to the promotion of the acceptance channel to current
workload: project_promoted:automate:acceptance:*
```